bundles/framework/layerlist/view/LayerViewTabs/LayerList/LayerCollapse/LayerCollapseHandler.js

```diff
@@ -237,8 +237,10 @@ class ViewHandler extends StateHandler {
         this.updateState({ selectedLayerIds });
     }
 
+    // Is this deprecated?
```

Comment:
This seems to also not be used anywhere, remove?

Reply:
It might be that the relevant logic has been moved to the MetadataIcon component. So if this is not called anywhere it can be removed.

Reply:
The only reference to the same named function is in admin-layereditor, where we have a different function with the same name (in AdminLayerFormHandler.js). I'll remove it.